xwiki-platform-tools/xwiki-platform-tool-configuration-resources/src/main/resources/xwiki.properties.vm

```diff
@@ -238,6 +238,83 @@ rendering.transformations = $xwikiRenderingTransformations
 #-#         \$sp and if you need a new line you'll need to use \$nl
 #-# rendering.macro.velocity.filter = indent
 
+#-------------------------------------------------------------------------------------
+# HTML Sanitization
+#-------------------------------------------------------------------------------------
+
+#-# [Since 13.10.7]
+#-# [Since 14.4.2]
+#-# [Since 14.5RC1]
+#-# The HTML sanitization strategy to use for user-generated content to avoid JavaScript injection. The following
+#-# strategies are available by default:
+#-# - secure (default): Only allows known elements and attributes that are considered safe. The following options
+#-#                     allow customizing its behavior.
+#-# - insecure:         Allows everything including JavaScript. Use this only if you absolutely trust everybody who can
+#-#                     write wiki syntax (in particular, all users, but also anonymous users commenting when enabled).
+# xml.htmlElementSanitizer = secure
+
+#-# [Since 13.10.7]
+#-# [Since 14.4.2]
+#-# [Since 14.5RC1]
+#-# Additional tags that should be allowed by the HTML sanitizer. These tags will be allowed in addition to the
+#-# already extensive built-in list of tags that are considered safe. Use with care to avoid introducing security
+#-# issues.
+# xml.htmlElementSanitizer.extraAllowedTags =
+
+#-# [Since 13.10.7]
+#-# [Since 14.4.2]
+#-# [Since 14.5RC1]
+#-# Additional attributes that should be allowed by the HTML sanitizer. These attributes will be allowed in addition to
+#-# the already extensive built-in list of attributes that are considered safe. This option is useful if your content
+#-# uses attributes that are invalid in HTML. Use with care to avoid introducing security issues.
+# xml.htmlElementSanitizer.extraAllowedAttributes =
+
+#-# [Since 13.10.7]
+#-# [Since 14.4.2]
+#-# [Since 14.5RC1]
+#-# Tags that should be forbidden even though they are initially allowed. This can be used, for example, to forbid
+#-# video or audio elements for unprivileged users.
+# xml.htmlElementSanitizer.forbidTags =
```

Comment:
What get the priority between allowedTags and forbidTags?

Reply:
`forbidTags` wins, I'll make this clearer, thank you for noticing this.